src/main/java/org/commonjava/util/partyline/lock/local/LocalLockOwner.java

```diff
@@ -182,36 +184,39 @@ synchronized boolean unlock( final String label )
                     this.dominantLockLevel = newDom.level;
                     logger.trace( "New dominant holder is: {} with level: {}", this.dominantOwner,
                                   this.dominantLockLevel );
+                    return new UnlockStatus( true, true, this.dominantLockLevel );
```

Comment:
I think it's still possible that the new dominant lock level is the same as the old one, but the old owner has been replaced by a new dominant owner...we should check that to be sure we're setting the isChanged field correctly.